### PR TITLE
STConfig: Separate stconfig commands into pkg stconfig

### PR DESCRIPTION
**Pull request by ChriMarMe**
Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

File: pkg/stconfig/sign.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package stconfig
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/u-root/u-root/pkg/bootconfig"
+)
+
+// AddSignatureToBootConfiguration TODO:
```
Comment (rjoleary) [inline pkg/stconfig/sign.go]: What is the TODO?

File: pkg/stconfig/keygen.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package stconfig
```
Comment (rjoleary) [inline pkg/stconfig/keygen.go]: To make it a bit more compact, can you put these four functions in one file `pkg/stconfig/stconfig.go`?

File: pkg/stconfig/keygen.go [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/u-root/u-root/pkg/crypto"
+)
+
+// GenKeys generates ED25519 keypair and stores it on the harddrive
```
Comment (rjoleary) [inline pkg/stconfig/keygen.go]:
```suggestion
// GenKeys generates ED25519 keypair and stores it on the filesystem
```

File: pkg/stconfig/keygen.go (hunk 2 above)
Comment (rjoleary) [inline pkg/stconfig/keygen.go]: Can you add a license to this file?
A godoc for the package would also be welcome.

File: pkg/stconfig/keygen.go [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+func GenKeys(genkeysPrivateKeyFile, genkeysPublicKeyFile, genkeysPassphrase string) error {
+	if _, err := os.Stat(genkeysPrivateKeyFile); os.IsNotExist(err) {
```
Comment (rjoleary) [inline pkg/stconfig/keygen.go]: I don't entirely understand this. The file must already exist before GenerateED25519Key creates it?

File: pkg/stconfig/unpack.go [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+// UnpackBootConfiguration unpacks a boot configuration file and returns the
+// file path of a directory containing the data
+func UnpackBootConfiguration(unpackInputFilename string) error {
+	_, outputDir, err := bootconfig.FromZip(unpackInputFilename)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Boot configuration unpacked into: " + outputDir)
```
Comment (rjoleary) [inline pkg/stconfig/unpack.go]: Prefer to use `log.Println` here.

**Comment by ChriMarMe** [comment]
This does not go into master, it is for stboot branch. I expected @jensdrenhaus to review this.

**Pull request closed without merging**